.github/workflows/release.yml

```
@@ -28,3 +28,11 @@ jobs:
           CF_API_KEY: ${{ secrets.CURSEFORGE_API_KEY }}
           WOWI_API_TOKEN: ${{ secrets.WOW_INTERFACE_API_KEY }}
           WAGO_API_TOKEN: ${{ secrets.WAGO_API_KEY }}
+
+      - name: Shotta Discord notification
+        env:
```

If this gets annoying for release candidate releases, we may want to move this into a separate workflow, or disable it with something like

```suggestion
      - name: Shotta Discord notification
        if: !contains(github.ref_name, "rc")
        env:
```